src/templates/sitegate/inline.html

```diff
@@ -10,11 +10,12 @@ <h2 class="pf-widget-headline"></h2>
             <p name="google-login" hidden></p>
           </div>
           <form>
-            <input class="pf-sitegate-field pf-field-full-width" name="username" type="text">
-            <input  class="pf-sitegate-field pf-field-full-width" name="email" type="email">
-            <input class="pf-sitegate-field pf-field-half-width" name="organization" type="text">
-            <input class="pf-sitegate-field pf-field-half-width" name="title" type="text">
-            <div class="pf-sitegate-clear"></div>
+            <input name="username" type="text">
+            <input name="email" type="email">
+            <input class="pf-field-half-width" name="company" type="text">
```

Comment:
If you turn off one, the other goes full width. Don't worry I accounted for it. :)